configure.ac

```diff
@@ -303,6 +303,16 @@ AC_CANONICAL_BUILD
 AC_CANONICAL_HOST
 AC_CANONICAL_TARGET
 
+# Check for the --with-android option
+AC_ARG_WITH([android],
+  [AS_HELP_STRING([--with-android], [enable Android support])],
+  [android=yes], [android=no])
+
+# If --with-android is specified, add -landroid-shmem to oc_ldflags
+# This in shea Allah ensures that it compiles within termux/android.
```

Comment:
Putting my French "laicité" hat on, I'm going to ask you to please remove "in shea Allah" here.  (Reference for other reviewers: https://en.wikipedia.org/wiki/Inshallah .)  I would ask the same had you written "may God help us".  Invoking God ins a source code comment is out of place.